web-common/src/features/dashboards/time-controls/TimeRangeSelector.svelte

```diff
@@ -221,18 +241,16 @@
         {allTime.label}
       </span>
     </MenuItem>
-    {#if $timeRangeSelectorState.showDefaultItem}
+    {#if $timeRangeSelectorState.showDefaultItem && defaultTimeRange}
```

Comment:
Hmm this check feels redundant since `showDefaultItem` already incorporates the check if `defaultTimeRange` is defined in the dashboard config.